tools/charset.py

```diff
@@ -15,7 +15,7 @@ def is_special_char(c):
 
 def get_special_chars():
   result = {}
-  for lang in["cn", "cz", "da", "de", "en", "es", "fi", "fr", "he", "it", "jp", "nl", "pl", "pt", "se", "tw"]:
+  for lang in["cn", "cz", "da", "de", "en", "es", "fi", "fr", "he", "it", "jp", "nl", "pl", "pt","ru","se", "tw"]:
```

Comment:
Minor cosmetic issue - spaces missing before and after "ru" here.

Reply:
cosmetic fix